liveblog/liveblog
# [LBSD-2712] - Live streams get paused

### Purpose

This PR fixes a bug that stops live streams from auto playing when loaded.

### What has changed

- Added `autoplay=1&mute=1` to the Youtube video URL to autoplay and mute the video. 
- Added `autoplay` to the iframe allow attributes

### Steps to test

1. Navigate to the Editor's interface.
2. Click "Add content here" and select Embed. Add a link to a live Youtube video and publish
3. Click "Live" to see the Live Preview. You will see the posted video which should then autoplay.

_Note: The video will autoplay when muted. This is because most modern web browsers have implemented strict policies around autoplaying videos in an effort to improve the user experience and provide greater control of playback to viewers._


Resolves: [LBSD-2712](https://sofab.atlassian.net/browse/LBSD-2712)

[LBSD-2712]: https://sofab.atlassian.net/browse/LBSD-2712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
